### PR TITLE
highlight system messages in search results

### LINKED ISSUE #1912: system messages not highlighted in search
- Steps to reproduce the problem: search for some text that match also system messages, the system messages are included in results but when you use the arrows to navigate through the results, normal messages are highlighted but system messages aren't and you don't have visual feedback.

### PR DESCRIPTION
**Pull request by adbenitez**
close #1912

### CODE REVIEW

File: src/org/thoughtcrime/securesms/ConversationUpdateItem.java [hunk 1]
```diff
@@ -55,9 +55,8 @@ public void bind(@NonNull DcMsg                   messageRecord,
                    @NonNull Recipient               conversationRecipient,
                             boolean                 pulseUpdate)
   {
-    bind(messageRecord, dcChat, batchSelected);
+    bind(messageRecord, dcChat, batchSelected, pulseUpdate);
     setGenericInfoRecord(messageRecord);
-    setSelected(batchSelected.contains(messageRecord));
```
Comment (Hocuri) [inline src/org/thoughtcrime/securesms/ConversationUpdateItem.java:60]: What does this change do?

**Review by Hocuri — APPROVED** [review]
Thanks!

Tested and looked through code

File: src/org/thoughtcrime/securesms/ConversationUpdateItem.java (hunk 1 above)
Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ConversationUpdateItem.java:60]: that is what adds support for the "pulse" effect in both normal messages and system messages, previously it was implemented only for normal messages, I moved that to the parent base class

Comment (Hocuri) [reply src/org/thoughtcrime/securesms/ConversationUpdateItem.java:60]: Do you know why the `setSelected(batchSelected.contains(messageRecord));` was there before?

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ConversationUpdateItem.java:60]: @Hocuri yes, it was to mark message as selected when you are in "selection mode" (to put the light blue background) that logic is inside `BaseConversationItem.setInteractionState()` which takes care of the interaction state (pulse effect and selected row)

**Pull request merged**